telemetry/block.go

```diff
@@ -0,0 +1,98 @@
+package telemetry
```

Comment:
This LGTM but I can't tell if you intended to remove it or not. 

Going to approve and up to you to decide on what's next.

Reply:
Removed the disk consumption metric which may impact the block production timings when the state size grows.

After talking to @bryanchriswhite, It seemed interesting to have a view on the submitted Txs size as well as the resulting block size. Since these metrics are not resource hungry, it's safe to leave them here and potentially removing them later if no-longer needed.